index.html

```diff
@@ -345,11 +353,17 @@ <h3>Supply Chain Import</h3>
           <li>X. Re-Presentation and VC+VP Verification across a common supply chain process.</li>
           </ul></p>
         <p><b>Endpoints:</b></li>
-          <li><i>(VERIFIER-APP?) /presentations/available,</i></li>
-          <li>/presentations/issue,</li>
-          <li>/presentations/submissions,</li>
-          <li>[/presentations/verify, /credentials/verify,]</li>
-          <li>/credentials/issue</li>
+          <li>Holder (SMG) --> Holder App --> Verifier App: Notify ( GET /presentations/available ) </li>
+          <li>Holder App <-- Verifier App: return Domain & Challenge </li>
+          <li> {Holder App --> Holder Storage: get VCs ( out of scope )}</li>
+          <li> {Holder App --> Holder Storage and/or KMS: get keys ( out of scope )}</li>
```

Comment:
I don't understand this one. Is it to the storage or to the KMS? And why are we getting keys? IMO, private keys should never be transmitted.
```suggestion
```

Reply:
it was in the diagram from your meeting with them, but omitted in the corresponding use-case.  i wasn't at the meeting so I have no context